pkg/reconciler/common/releases.go

```diff
@@ -213,6 +237,20 @@ func FetchManifest(path string) (mf.Manifest, error) {
 	return result, err
 }
 
+// FetchManifestFromPath returns the manifest by reading them from the path, and saves them in the cache.
+func FetchManifestFromPath(path string) (mf.Manifest, error) {
```

Comment:
```suggestion
func fetchManifestFromPath(path string) (mf.Manifest, error) {
```

Can you make this function private? It is not called from other packages.